README.md

```diff
@@ -213,3 +213,15 @@ image placeholder
 
 ### Solution
 Lab challenge (needs a whiteboard after class)
+
+---
+### Challenge 21: Insertion sort
+Sorts an array from left to right iteratively placing each new element into the appropriate index.
+
+### Approach and Efficiency
+At it’s worst, the array is in the reverse order, giving a quadratic runtime of O(n^2), but on average it isn't the worst possible runtime so it will still be O(n^2) with any data set.

```

Comment:
You haven't noted if your Big O analysis if it is space or time and you haven't specified an analysis for both.